src/plugins/home/public/application/components/usecase_overview/search_use_case_setup.tsx

```diff
@@ -0,0 +1,252 @@
+      title: i18n.translate('home.searchOverview.setup.createDocumentIndex.title', {
+        defaultMessage: 'Create a document index',
+      }),
+      description: i18n.translate('home.searchOverview.setup.createDocumentIndex.description', {
+        defaultMessage:
+          'You can create a document collection (an index) by adding documents to a new index.',
+      }),
+      footer: (
+        <EuiI18n
+          token="home.search_overview.createDocumentIndex.card.footer"
+          default="Documentation"
+        />
+      ),
+      documentURL: 'https://opensearch.org/docs/latest/getting-started/intro/',
+    },
+  ];
+
+  getStartedCards.forEach((card) => {
+    contentManagement.registerContentProvider({
+      id: card.id,
+      getContent: () => ({
+        id: card.id,
+        kind: 'card',
+        order: card.order,
+        title: card.title,
+        description: card.description,
+        cardProps: {
+          selectable: {
+            onClick: () => {
+              window.open(card.documentURL, '_blank');
+            },
+            children: card.footer,
+            isSelected: false,
+          },
+        },
+      }),
+      getTargetArea: () => SEARCH_OVERVIEW_CONTENT_AREAS.GET_STARTED,
+    });
+  });
+
+  const searchIcon = <EuiIcon color="subdued" size="l" type="search" />;
+
+  const searchTypeCards = [
+    {
+      id: 'text_search',
+      order: 10,
+      title: i18n.translate('home.searchOverview.searchTypes.textSearch.title', {
+        defaultMessage: 'Text search',
+      }),
+      description: i18n.translate('home.searchOverview.searchTypes.textSearch.description', {
+        defaultMessage:
+          'Lexical or keyword search matches documents based on exact words or phrases. Search the text using human-friendly query string query syntax or create complex, customizable queries using Query DSL—the OpenSearch query language.',
+      }),
+      icon: searchIcon,
+      footer: (
+        <EuiLink
+          external
+          target="_blank"
+          href="https://opensearch.org/docs/latest/query-dsl/full-text/query-string/"
+        >
+          View Documentation
+        </EuiLink>
+      ),
+    },
+    {
+      id: 'analyzer',
+      order: 10,
+      title: i18n.translate('home.searchOverview.searchTypes.analyzer.title', {
+        defaultMessage: 'Analyzers',
+      }),
+      description: i18n.translate('home.searchOverview.searchTypes.analyzer.description', {
+        defaultMessage:
+          'Analyzers prepare text for indexing. For example, HTML text typically has its tags removed. English text typically treats paint, painted, and painting as equivalent by mapping them all to the token paint. You can use OpenSearch’s extensive library of standard analyzers or create your own.',
+      }),
+      icon: searchIcon,
+      footer: (
+        <EuiLink
+          external
+          target="_blank"
+          href="https://opensearch.org/docs/latest/analyzers/search-analyzers/"
```

Comment:
Shall we use/add these links to `doc_links_service` in `src/core/public/doc_links/doc_links_service.ts`?

Reply:
good suggestion, we can have a follow up PR to add those links into doc_link_services.